packages/esm-service-queues-app/src/overlay.scss

```diff
@@ -11,8 +11,8 @@
   border-left: 1px solid $text-03;
   background-color: $ui-01;
   overflow-y: auto;
-  height: 100vh;
-  z-index: 99999;
+  height: calc(100vh - 3rem);
+  z-index: 3;
```

Comment:
This is an unrelated issue, but has been bothering me for a while, in that the buttons in the Add to queue overlay are barely visible on the screen.  This adjustment here I copied from the appointments esm, which presumably had the same issue and was fixed there.  Yet another reason not to have multiple overlay implementations all over the place.